lib/index.js

```diff
@@ -214,8 +214,8 @@ Segment.prototype.normalize = function(msg) {
     }
   }
   // if user provides campaign via context, do not overwrite with UTM qs param
-  if (query && !ctx.campaign) {
-    ctx.campaign = utm(query);
+  if (!ctx.campaign) {
+    ctx.campaign = query ? utm(query): {}
```

Comment:
```suggestion
    ctx.campaign = query ? utm(query) : {};
```

🐜  just to keep the coding style similar with the rest of the code

Reply:
@jopdeklein nice thank you!